core/model/base_user.py

```diff
@@ -42,27 +42,16 @@ def parametrizer(self):
     def fields(self):
         return [
             Field("counters", Counters),
-            Field(
-                "last_action_ids",
-                LimitedQueuedHashableObjectsItems,
-                self.descriptions["last_action_ids"]
-            ),
-            Field(
-                "last_messages_ids",
-                LimitedQueuedHashableObjects,
-                LimitedQueuedHashableObjectsDescription(None)
-            ),
+            Field("last_action_ids", LimitedQueuedHashableObjectsItems, self.descriptions["last_action_ids"]),
+            Field("last_messages_ids", LimitedQueuedHashableObjects, LimitedQueuedHashableObjectsDescription(None)),
             Field("local_vars", Variables, None, False),
             Field("variables", Variables),
         ]
 
-    @property
-    def raw(self):
-        return super(BaseUser, self).raw
-
     @property
     def raw_str(self):
-        raw = json.dumps(self.raw)
+        # Attention: non-serializable objects will become str with error message
+        raw = json.dumps(self.raw, default=lambda o: f"<non-serializable: {type(o).__qualname__}>")
```

Comment:
зочем дефолт? оно потом не загрузится из этой строки

Reply:
А надо вообще скипать? Для этого придётся либо заранее проходить по объекту, убирая непиклимые объекты, либо после, иначе нельзя, потому что у json.dumps нет аргумента со скипом.

Reply:
как это нет?
If skipkeys is true (default: False), then dict keys that are not of a basic type ([str](https://docs.python.org/3/library/stdtypes.html#str), [int](https://docs.python.org/3/library/functions.html#int), [float](https://docs.python.org/3/library/functions.html#float), [bool](https://docs.python.org/3/library/functions.html#bool), None) will be skipped instead of raising a [TypeError](https://docs.python.org/3/library/exceptions.html#TypeError).

Reply:
Это игнорируются непиклимые ключи, для значений есть только дефолт